test/unit/single_template_test.rb

```
@@ -0,0 +1,378 @@
+require_relative "../base_page_test_case"
+
+class SingleTemplateTest < BasePageTestCase
+  # Comprehensive tests for single.html template
+  # Validates individual post/page functionality, content structure, and SEO
+  # Implements TDD coverage per /knowledge/20.01-tdd-methodology-reference.md
+
+  def setup
+    # Test with a known blog post or page
+    @test_pages = [
+      "blog/index.html",
+      "about/index.html"
```

RuboCop (1.80.2)

[convention] Put a comma after the last item of a multiline array.

(Style/TrailingCommaInArrayLiteral)